process/incubation.md

```diff
@@ -39,8 +41,27 @@ For a pitch to be moved into the Proposal stage, it must be endorsed by at least
 * Social media accounts (optional)
 * Community size and any existing sponsorship
 
+Once a propsal PR is submitted, the SSWG will assign a review manager during it's bi-weekely meeting. The review manager responsibilities include:
+
+* Review the PR 
+  * Vaidate structure and language.
+  * Make sure the implementation is in place.
+* Update the PR
+  * Assign a number.
+  * Assign a review manager.
+  * Set status to "Active Review + Date range for review"
+* Secure approval from the PR author to merge changes above and merge them.
+* Publisha a form post in the [server proposals area](https://forums.swift.org/c/server/proposals) soliciting feedback from the community.
+* Watch the forum thread for conduct or off-topic issues, and make sure author is engaged.
+* Once the review period is over, summarize the key takeaways to the SSWG in writing.
```

Comment:
```suggestion
Once a Proposal PR is submitted, the SSWG will assign a review manager during it's bi-weekly meeting.

The review manager responsibilities include:
* Reviewing the PR:
  * Validating structure and language
  * Ensuring the implementation is "complete", as defined by the Pitch phase
* Updating the PR:
  * Assigning a Proposal number
  * Assigning the review manager
  * Updating the status to "Active Review (Date range for review)"
* Coordinating with the PR author to merge the updated PR details into the Proposal document
* Publishing a form post in the [server proposals area](https://forums.swift.org/c/server/proposals) to request implementation feedback from the community
```

Reply:
we ost L56?

Reply:
No, not intentional. Still getting used to GitHub's multi-line suggestions and those lines were lost.

Updated my suggestion to include those lines